.github/workflows/main.yml

```diff
@@ -4,45 +4,42 @@ name: CI
 
 # Controls when the workflow will run
 on:
-  # Triggers the workflow on push or pull request events but only for the master branch
   push:
     branches: [ master ]
   pull_request:
     branches: [ master ]
-
-  # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  # This workflow contains a single job called "build"
   build:
-    # The type of runner that the job will run on
     runs-on: ubuntu-latest
 
-    # Steps represent a sequence of tasks that will be executed as part of the job
+    strategy:
+      matrix:
+        # List of actively supported Node.js LTS releases
+        # Check https://nodejs.org/en/about/releases for the current versions and update the following if needed:
+        node-version: [18.x, 20.x, 22.x]
```

Comment:
Do we **really** need to run the test suite against 3 node version?
Most of the node code is really dump, and is not likely to broke because of node version.

More over, many tests are run by other language runtime

IMHO, it's just a waste of resource 🌍 🔥

Reply:
Sure, it has been a habit from my other projects since I had past issues with specific versions of node. I see your point and reverted the matrix run.